Formula/packs.rb

```diff
@@ -4,7 +4,7 @@
 class Packs < Formula
   desc "A pure Rust implementation of packwerk, a gradual modularization tool for Ruby"
```

Comment:
May as well fix the error here while we are at it

```suggestion
  desc "Pure Rust implementation of packwerk, a gradual modularization tool for Ruby"
```